Polyhedron/demo/Polyhedron/Plugins/IO/CMakeLists.txt

```
@@ -80,6 +80,7 @@ endif()
 
 find_path(3MF_INCLUDE_DIR
   NAMES Model/COM/NMR_DLLInterfaces.h
+  PATHS /usr/include/lib3mf
```

Instead of hardcoding `/usr/include`, you can use:
```suggestion
  PATH_SUFFIXES lib3mf
```